<ui version="4.0" >
 <class>uorTextFontProp</class>
 <widget class="QWidget" name="uorTextFontProp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>113</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>1</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>360</width>
    <height>102</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>2</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>2</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item row="0" column="0" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>2</number>
       </property>
       <property name="spacing" >
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Font</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_fontName" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>130</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="m_listFonts" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>1</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="sizeIncrement" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>160</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet" >
          <string>QListWidget { show-decoration-selected: 1;}</string>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>2</number>
       </property>
       <property name="spacing" >
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Size</string>
           </property>
           <property name="openExternalLinks" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_size" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="m_sizeList" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>1</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement" >
          <size>
           <width>0</width>
           <height>1</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>50</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet" >
          <string>QListWidget { show-decoration-selected: 1;}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>2</number>
       </property>
       <property name="spacing" >
        <number>2</number>
       </property>
       <item>
        <widget class="QCheckBox" name="m_chbItalic" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Italic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_chbUnderline" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Underline</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chbBold" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Bold</string>
         </property>
        </widget>
       </item>
         <item>
          <widget class="QLabel" name="labelColor" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Color</string>
           </property>
           <property name="buddy" >
            <cstring>m_cBoxColor</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="m_colFrm" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>1</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth" >
            <number>0</number>
           </property>
          </widget>
         </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_fontName</tabstop>
  <tabstop>m_listFonts</tabstop>
  <tabstop>m_size</tabstop>
  <tabstop>m_sizeList</tabstop>
  <tabstop>m_chbItalic</tabstop>
  <tabstop>m_chbUnderline</tabstop>
  <tabstop>chbBold</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
